lib/syskit/roby_app/plugin.rb

```diff
@@ -250,6 +250,18 @@ def self.cleanup(app)
             # Hook called by the main application to prepare for execution
             def self.prepare(app)
                 @handler_ids = plug_engine_in_roby(app.execution_engine)
+
+                @log_upload_command_queue = Queue.new
+                @log_upload_thread = Thread.new { 
+                    app.log_upload_main(@log_upload_command_queue)
+                }
+                @rotated_logs = {}
```

Comment:
Why is that a thread ? The process servers already handle this asynchronously, aren't they ?